tests/Fixtures/functions/template_from_string_error.test

```diff
@@ -1,5 +1,7 @@
 --TEST--
 "template_from_string" function
+--CONDITION--
+version_compare(phpversion(), '8.1.0-dev', '<')
```

Comment:
I'd rather have the suffixed file name for old PHP versions than for new ones (the day the min version is bumped to PHP 8.1+, we would then keep only the non-suffixed file)

Reply:
That's pertinent. I renamed the files.

Reply:
btw, why not using the same `PHP_VERSION_ID` condition ?

Reply:
Because I blindly followed other fixtures. Fixed.